src/main/osd/osd_elements.c

```diff
@@ -992,6 +992,15 @@ static void osdElementFlymode(osdElementParms_t *element)
     }
 }
 
+static void osdElementReadyMode(osdElementParms_t *element)
+{
+    if (IS_RC_MODE_ACTIVE(BOXREADY)) {
+        if (calculateThrottlePercent() < 0.01) {
```

Comment:
Is this safe? It will flash 'READY' when throttle is zero after takeoff

calculateThrottlePercent() returns `uint8_t` ( https://github.com/betaflight/betaflight/blob/master/src/main/fc/core.c#L742) so this test is confusing and should be  `calculateThrottlePercent() < 1`

Reply:
Maybe the test should be boxReady && armed ?

Reply:
Won't harm (is pilot ready before arming? IMO it depends on rules / use case).
But "READY" is only dependent on throttle, so it will appear each time when throttle is zero during flight (+ in 3d mode). It may be disturbing ...

Reply:
I will add a condition of takeoff that resets on disarm so it doesn't flash